core/replica.go

```diff
@@ -19,34 +19,18 @@ package minbft
 
 import (
 	"fmt"
-	"os"
 
 	"github.com/golang/protobuf/proto"
-	logging "github.com/op/go-logging"
 
 	"github.com/hyperledger-labs/minbft/api"
 	"github.com/hyperledger-labs/minbft/core/internal/messagelog"
 )
 
 const (
-	module          = "minbft"
-	logFormatString = `%{color}[%{module}] %{time:15:04:05.000} %{shortfunc} ▶ %{level:.4s} %{id:03x}%{color:reset} %{message}`
+	module           = "minbft"
+	defaultLogPrefix = `%{color}[%{module}] %{time:15:04:05.000} %{shortfunc} ▶ %{level:.4s} %{id:03x}%{color:reset}`
```

Comment:
The name `defaultLogPrefix` implies that this will be configurable at some time, so I'll create an issue for this later.
And I think that `client/client.go` can also have some prettier logging.